### ISSUE TITLE
zsh terminal command competition adds extra characters at start

### ISSUE THREAD

**Issue by ashwinmenkudle** [cdr/code-server#4377]
## OS/Web Information

- Web Browser: Chrome
- Local OS: Windows 10
- Remote OS: Ubuntu
- Remote Architecture:
- `code-server --version`: 3.11.1 c680aae973d83583e4a73dc0c422f44021f0140e

## Steps to Reproduce

1. Open a terminal with zsh
2.  Installed ohmyzsh
3. Try any command with autocompletion with tab.
4. Extra characters get added at the start.

## Expected

Extra characters should not get added at the start.
It is possible to edit the command after command completion.

## Actual

Extra characters get added at the start.

## Screenshot

![issue](https://user-images.githubusercontent.com/5318013/137893694-de17dc40-9660-4f1e-b9b8-e298ba67278b.gif)

This issue can be reproduced in VS Code: No

**Comment by ashwinmenkudle** [comment]
> If you have SSH access can you verify it does not happen in a regular terminal?

Yes, Autocomplete works fine with mobaxterm ssh client with zsh shell and bash shell.
Basic autocomplete supported by bash shell also works correctly in code server.

**Comment by jsjoeio** [comment]
@echuber2 thank you for following up and confirming! And thanks for posting that user settings snippet workaround (if the issue still happens). In that case, I'm doing to close this for now. If it still happens for others, comment here and we'll reopen.
